frontend/public/locales/en/translation.json

```diff
@@ -40,6 +40,7 @@
   "<0>Run this command</0> to log-in to the existing cluster in your terminal": "<0>Run this command</0> to log-in to the existing cluster in your terminal",
   "<bold>Best practice:</bold> Use Placement.": "<bold>Best practice:</bold> Use Placement.",
   "<bold>Disabled:</bold> The automation does not run automatically.": "<bold>Disabled:</bold> The automation does not run automatically.",
+  "<bold>Note:</bold> Data from the last snapshot taken will be lost. To prevent losing current data, take another snapshot before restoring from this one.": "<bold>Note:</bold> Data from the last snapshot taken will be lost. To prevent losing current data, take another snapshot before restoring from this one.",
```

Comment:
For a long string with tags in it, I would consider using a symbolic translation key. For the string "Data from the last snapshot taken will be lost" I think it might be clearer to say something like "Changes to data made after the last snapshot was taken will be lost."